config.yaml

```diff
@@ -0,0 +1,35 @@
+linkerdVersion: stable-2.8.0
+externalIssuer: false
+controlPlane:
+    # namespace: l5d-conformance
+    config:   
+        ha: false
+        flags:
+            - "--controller-log-level"
+            - "debug"
+            - "--proxy-log-level"
+            - "warn,linkerd2_proxy=debug"
```

Comment:
Do these config apply to all, most or specify test cases? I'd think something like `ha`, `flags` are only relevant to control plane installation tests. Will it make sense to group them under the `lifecycle` configuration?

Reply:
These config apply to all installations of linkerd. Currently, we support to ways of control plane installation : 
1. a "universal" control plane - the control plane is installed as a part of the "lifecycle test", and the same control plane installation is used for the remaining tests.
2. a "local" control plane - the control plane is installed and uninstalled as a part of the "lifecycle" tests. Each suceeding test will then install a control plane, run the tests and assertions, and finally uninstall once the test is complete.

These configs are common to both our above cases. I'm completely okay with moving these under "lifecycle". @Pothulapati wdyt?

Reply:
Hmm, Here we wanted to support both the cases where a test would and would not need a specific linkerd installation. If a specific test would need a different installation, the test-case can perform its own right in a different namespace, WDYT? :thinking: